src/styles/index.scss

```diff
@@ -1,3 +1,71 @@
+html {
+  height: 100%;
+}
+
 body {
   margin: 0;
+  height: 100%;
```

Comment:
You have `height: 100%;` in both the `html` and `body` selectors. This is not necessary. You can remove the `height: 100%;` from the `body` selector.

Reply:
this is needed for vertical centering